client/src/components/Q&A/AddQModal.jsx

```diff
@@ -23,10 +23,30 @@ export default function AddQModal({ show, setShow }) {
       </div>
       <div className="modal-content">
         <div className="modal-body">
-          <h4>Modal</h4>
-          <p>Some words to read</p>
+          <div className="modal-top-info">
+            <h3>Nickname</h3>
+            <h3>Email</h3>
+          </div>
+          <div className="modal-nickname-email">
+            <input type="text" placeholder="Example: jackson11!" />
+            <input type="text" placeholder="aperson@gmail.com" />
+          </div>
+          <div className="modal-info">
+            <span>For privacy reasons, do not use your full name or email address</span>
+            <span>For authentication reasons, you will not be emailed</span>
+          </div>
+          <textarea
+            className="modal-input"
+            placeholder="Why did you like the product or not?"
```

Comment:
I know the docs probably say that this should be the placeholder, but maybe something like "Ask your question here!", We can discuss it as a group, but seems like we are asking for a review with the current placeholder.

Reply:
Ahh! I was looking at the wrong section in the business req's!